### ISSUE TITLE
Application doesn't want to open anymore and just shows window icon with no actual window

### ISSUE THREAD

**Issue by TheCrimsonSage** [IsmaelMartinez/teams-for-linux#278]
**Describe the bug**
The application doesn't want to open, it just shows the window and tray icon but there is no actual window. Even if I press the Windows key I can't see the window but there is an open window icon in my dock. I have deleted the config and cache folders multiple times, I've tried to run it as sudo, nothing works. I've uninstalled and reinstalled different channels of the app (.deb, AppImage, snap) and I get the same results aside from the snap version not starting at all. It used to work with the previous version but just suddenly stopped working and then I downloaded the new version but it's still not working.

**To Reproduce**
1. Install teams-for-linux from .deb, AppImage, etc.
2. No Window at all

![image](https://user-images.githubusercontent.com/19888926/64508319-7855b900-d2dd-11e9-9720-fa0e129e2707.png)

**Desktop (please complete the following information):**
 - OS: teams-for-linux-0.7.0.AppImage
 - Installation package deb and AppImage
 - 0.70

**Comment by TheCrimsonSage** [comment]
Also worth noting that I've tried the refresh and debug buttons for the tray icon but they all have the same symptoms

**Comment by KPS-MWT** [comment]
Can you give us your exact OS (e.g. DEbian, Fedora), Version of OS and used Desktop (looks like Gnome, but which version?)

**Comment by IsmaelMartinez** [comment]
Yeah, as @KPS-MWT says, it will be great if you can provide as much details as you can from  the bug template. It helps us helping you to find the solution, and others to know if they are experiencing the same issue. 

also, what do you see if you run it from the console? 

```bash
teams-for-linux
```

Thanks.

**Comment by TheCrimsonSage** [comment]
Sorry my bad, I thought I included the information. 
OS: Ubuntu 19.04 64-bit
Gnome version: 3.32.1

**Comment by TheCrimsonSage** [comment]
Thank you guys for your help. When I run it from the terminal as a regular user I get the following output even though I deleted the config:
```
configPath = /home/jason/.config/teams-for-linux
Failed to get the config file, using default values
configFile = {}
```
But then I can confirm that I can see the files being recreated when I launch it
![image](https://user-images.githubusercontent.com/19888926/64534009-f2edfb00-d314-11e9-94a3-bbaeb085db00.png)

**Comment by TheCrimsonSage** [comment]
If I run it as sudo I get the following output:
```
configPath = /home/jason/.config/teams-for-linux
Failed to get the config file, using default values
configFile = {}

(teams-for-linux:22197): libappindicator-WARNING **: 15:18:39.193: Unable to get the session bus: Unknown or unsupported transport “disabled” for address “disabled:”

(teams-for-linux:22197): LIBDBUSMENU-GLIB-WARNING **: 15:18:39.193: Unable to get session bus: Unknown or unsupported transport “disabled” for address “disabled:”
```

**Comment by IsmaelMartinez** [comment]
can you see the process running? (`ps -aux`).

**Comment by TheCrimsonSage** [comment]
It seems to be running when I check `ps -aux | grep teams`
```
jason    22688  1.2  0.7 870200 114940 pts/0   Sl+  15:21   0:00 /opt/teams-for-linux/teams-for-linux
jason    22690  0.0  0.2 207544 39928 pts/0    S+   15:21   0:00 /opt/teams-for-linux/teams-for-linux --type=zygote --no-sandbox
jason    22814  0.2  0.4 306376 67188 pts/0    Sl+  15:21   0:00 /opt/teams-for-linux/teams-for-linux --type=gpu-process --enable-features=SharedArrayBuffer --no-sandbox --gpu-preferences=KAAAAAAAAACAAABAAQAAAAAAAAAAAGAAAAAAAAAAAAAIAAAAAAAAAAgAAAAAAAAA --service-request-channel-token=16407322983795272819
jason    22820  0.1  0.4 490056 76164 pts/0    Sl+  15:21   0:00 /opt/teams-for-linux/teams-for-linux --type=renderer --no-sandbox --enable-features=SharedArrayBuffer --service-pipe-token=16943917556377567986 --lang=en-GB --app-path=/opt/teams-for-linux/resources/app.asar --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) teams-for-linux/0.7.0 Chrome/69.0.3497.128 Electron/4.2.8 Safari/537.36 --node-integration=true --webview-tag=true --no-sandbox --background-color=#fff --num-raster-threads=4 --enable-main-frame-before-activation --service-request-channel-token=16943917556377567986 --renderer-client-id=4 --shared-files=v8_context_snapshot_data:100,v8_natives_data:101
jason    22868  0.6  0.6 551400 105448 pts/0   Sl+  15:21   0:00 /opt/teams-for-linux/teams-for-linux --type=renderer --no-sandbox --enable-features=SharedArrayBuffer --service-pipe-token=1008310057181374636 --lang=en-GB --app-path=/opt/teams-for-linux/resources/app.asar --user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) teams-for-linux/0.7.0 Chrome/69.0.3497.128 Electron/4.2.8 Safari/537.36 --enable-plugins --node-integration=false --webview-tag=false --no-sandbox --native-window-open --preload=/opt/teams-for-linux/resources/app.asar/app/browser/index.js --background-color=#fff --num-raster-threads=4 --enable-main-frame-before-activation --service-request-channel-token=1008310057181374636 --renderer-client-id=7 --shared-files=v8_context_snapshot_data:100,v8_natives_data:101
jason    23014  0.0  0.0   8860   972 pts/1    S+   15:22   0:00 grep --color=auto teams
```

**Comment by IsmaelMartinez** [comment]
Can you try to run from source? Not sure what else to look into. Does smell like OS config. Can you otherwise try in a VM? Ta

**Comment by TheCrimsonSage** [comment]
Sorry for the delay in coming back to you, life got a little in the way. I have good news and bad news. Good news is that it's working again, the bad news is I don't know why. So what I did differently is that I searched in my entire home folder for anything teams related and it seems like there were some cache files (chrome, snap information and a few other folders that wasn't in the config folder, that I ended up deleting) and I used apt to purge everything for the teams install and now it's working. Sorry for wasting your time but I'm so glad it's working now. Thank you for all your hard work :)
